lib/google-apis/auth-client.js

```diff
@@ -69,6 +81,83 @@ function AuthClient(projectId, shouldReportErrors, key) {
 }
 
 /**
+ * Adds the given callback as a listener to the `init` event which will be
+ * broadcast once the client has inited the underlying transport and generated
+ * or failed to generate a token.
+ * @function addInitListener
+ * @param {Function} callback - the listener to callback on when the client has
+ *  inited
+ * @returns {Undefined} - does not return anything
+ */
+AuthClient.prototype.addInitListener = function(callback) {
+
```

Comment:
Actually, it would be preferable to not have this single line function in the first place. `_eventEmitter` is a pretty descriptive name and it `_eventEmitter.on('init'...)` is just as readable. Less boilerplate too.